src/main/java/com/entropyteam/entropay/employees/dtos/ClientDto.java

```diff
@@ -23,11 +23,12 @@ public record ClientDto(UUID id,
                         String preferredCurrency,
                         @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss") LocalDateTime createdAt,
                         @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss") LocalDateTime modifiedAt,
-                        String internalId) {
+                        String internalId,
+                        boolean active) {
```

Comment:
i think active should be mandatory. @NotNull(message = "Active state is mandatory")